Project-Aurora/Project-Aurora/Profiles/Discord/Control_Discord.xaml.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Aurora.Profiles.Discord.GSI;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+
+using System.IO;
+
+namespace Aurora.Profiles.Discord
+{
+    /// <summary>
+    /// Interaction logic for Control_Minecraft.xaml
+    /// </summary>
+    public partial class Control_Discord : UserControl {
+
+        private Application profile;
+
+        public Control_Discord(Application profile) {
+            this.profile = profile;
+
+            InitializeComponent();
+            SetSettings();
+            
+
+            profile.ProfileChanged += (sender, e) => SetSettings();
+        }
+
+        private void SetSettings() {
+            GameEnabled.IsChecked = profile.Settings.IsEnabled;
+        }
+
+        private void GameEnabled_Checked(object sender, RoutedEventArgs e) {
+            if (IsLoaded) {
+                profile.Settings.IsEnabled = GameEnabled.IsChecked ?? false;
+                profile.SaveProfiles();
+            }
+        }
+
+        private void PatchButton_Click(object sender, RoutedEventArgs e)
+        {
+            InstallPlugin();
+        }
+
+        private void UnpatchButton_Click(object sender, RoutedEventArgs e)
+        {
+            UninstallPlugin();
+        }
+
+        private void InstallPlugin()
+        {
+            string appdata = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+            string bd = Path.Combine(appdata, "BetterDiscord", "plugins", "AuroraGSI.plugin.js");
+
+            if (!File.Exists(bd))            
+                Directory.CreateDirectory(Path.GetDirectoryName(bd));           
+
+            using (FileStream pluginStream = File.Create(Path.Combine(bd)))
```

Comment:
Why would you combine the already combined path again?

Reply:
This was probably a leftover from something i was doing earlier, just changed it.